test/unit/Heatmap/GMUGradientTest.swift

```diff
@@ -18,38 +18,38 @@ import XCTest
 
 class GMUGradientTest: XCTestCase {
   
-  var gradient          : GMUGradient!
-  var kGradientColor    : [UIColor]!
-  var kStartPoints      : [NSNumber]!
-  var kColorMapSize     : UInt!
+  private var gradientColor    : [UIColor]!
+  private var startPoints      : [NSNumber]!
+  private var colorMapSize     : UInt!
   
   override func setUp() {
-    kGradientColor = [
+    super.setUp()
+    gradientColor = [
         UIColor(red: 102.0 / 255.0, green: 225.0 / 255.0, blue: 0, alpha: 1),
         UIColor(red: 1.0, green: 0, blue: 0, alpha: 1)
     ]
-    kStartPoints = [NSNumber(value: 0.2), NSNumber(value: 1.0)]
-    kColorMapSize = 3
+    startPoints = [NSNumber(value: 0.2), NSNumber(value: 1.0)]
+    colorMapSize = 3
   }
   
   override func tearDown() {
-    gradient = nil
-    kGradientColor = nil
-    kStartPoints = nil
-    kColorMapSize = nil
+    super.tearDown()
+    gradientColor = nil
+    startPoints = nil
+    colorMapSize = nil
   }
   
   func testInitWithColors() {
-    gradient = GMUGradient(colors: kGradientColor, startPoints: kStartPoints, colorMapSize: kColorMapSize)
+    let gradient = GMUGradient(colors: gradientColor, startPoints: startPoints, colorMapSize: colorMapSize)
     XCTAssertEqual(gradient.colors.count, gradient.startPoints.count)
   }
   
   func testInitWithEmptyColors() {
     do {
-      try GMUObectiveCTestHelper.catchObjectiveCException{
-        _ = GMUGradient(colors: [], startPoints: self.kStartPoints, colorMapSize: self.kColorMapSize)
+      try GMUObectiveCTestHelper.catchObjectiveCException {
+        _ = GMUGradient(colors: [], startPoints: self.startPoints, colorMapSize: self.colorMapSize)
       }
-    } catch let error{
+    } catch let error {
       XCTAssertNotNil(error)
```

Comment:
I think we should check the specific thrown error:

```swift
catch let error as NSError {
    XCTAssertEqual("NSInvalidArgumentException", error.domain)
}
```

Same comment across other `XCTAssertNotNil` checks in this test.